config/initializers/devise.rb

```diff
@@ -234,6 +234,9 @@
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
 
+  config.omniauth :github, ENV['GITHUB_KEY'], ENV['GITHUB_SECRET'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.